src/db/seeders/20200225213741-two_way_trip.js

```diff
@@ -8,19 +8,20 @@ module.exports = {
     gender: 'male',
     email: 'william.ishimwe@andela.com',
     passport: 'null',
-    password: process.env.MANAGER_PSWD,
+    password: '$2b$10$SO7hwwylEnHKZIv68w/ePOK4yybPnhXa5.tQSUBe.PSb6FBKyuVO.',
```

Comment:
Why did you remove env?

Reply:
There was trouble verifying that password fetched from .env and pasting that hashed password directly into testing seeds solved the issue.